yarn-project/sequencer-client/src/block_builder/solo_block_builder.test.ts

```diff
@@ -241,7 +244,7 @@ describe('sequencer/solo_block_builder', () => {
   describe('mock simulator', () => {
     beforeEach(() => {
       // Create instance to test
-      builder = new SoloBlockBuilder(builderDb, vks, simulator, prover);
+      builder = new SoloBlockBuilder(builderDb, vks, simulator, prover, Fr.ZERO, Fr.ZERO);
```

Comment:
is there a reason for not using the constants you defined earlier: `chainId, version`?

Reply:
Simple.
![Pop Tv Idiot GIF by Big Brother After Dark](https://media4.giphy.com/media/QA6NycQS3gassP111A/200.gif)